api/v1/handlers.go

```diff
@@ -1743,37 +1730,58 @@ func (s *apiV1) handleGetSystemConfig(c echo.Context, u *util.User) error {
 }
 
 type minerResp struct {
-	Addr            address.Address `json:"addr"`
-	Name            string          `json:"name"`
-	Suspended       bool            `json:"suspended"`
-	SuspendedReason string          `json:"suspendedReason,omitempty"`
-	Version         string          `json:"version"`
+	Addr            address.Address       `json:"addr"`
+	Name            string                `json:"name"`
+	Suspended       bool                  `json:"suspended"`
+	SuspendedReason string                `json:"suspendedReason,omitempty"`
+	Version         string                `json:"version"`
+	ChainInfo       *miner.MinerChainInfo `json:"chain_info"`
 }
 
 // handleAdminGetMiners godoc
 // @Summary      Get all miners
-// @Description  This endpoint returns all miners
-// @Tags         public,net
+// @Description  This endpoint returns all miners. Note: value may be cached
+// @Tags         admin,net
 // @Produce      json
 // @Success      200  {object}  string
```

Comment:
I think this should say `// @Success 200 {object} minerResp` instead (better annotations)